### PR TITLE
Added logs when test cases are passed.

### PR DESCRIPTION
**Pull request by Deepakchowdavarapu**
## Related Issue
- Current test pipelines are not showing logs when all tests are passed
Closes: #2156

#### Describe the changes you've made
Modified the custom GitHub Action (tester) to always capture and display logs, regardless of test outcome. Updated the action.yml file to include a new step that consistently outputs test logs.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Please let us know if any test cases are added
No new test cases were added. This change affects how existing tests are reported, improving visibility of test execution for both successful and failed runs.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

## Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by Deepakchowdavarapu** [comment]
@Sarthak160 please review

**Comment by Sarthak160** [comment]
@Deepakchowdavarapu, Please sign the CLA, Also check your changes are breaking the current test pipline.

**Comment by Sarthak160** [comment]
Also when I am seeing the logs, I am still not able to see the full logs of the passed testset.

**Comment by Deepakchowdavarapu** [comment]
recheck

**Comment by Deepakchowdavarapu** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/keploy/keploy/tree/main/.github/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> **2** out of **3** committers have signed the CLA.✅ @Sonichigo✅ @Sarthak160❌ @DeepakchowdavarapuYou can retrigger this bot by commenting **recheck** in this Pull Request

recheck

**Comment by Deepakchowdavarapu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Deepakchowdavarapu** [comment]
recheck

**Comment by Deepakchowdavarapu** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/keploy/keploy/tree/main/.github/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> **2** out of **3** committers have signed the CLA.✅ @Sonichigo✅ @Sarthak160❌ @DeepakchowdavarapuYou can retrigger this bot by commenting **recheck** in this Pull Request

recheck

**Comment by Deepakchowdavarapu** [comment]
recheck

**Comment by Deepakchowdavarapu** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/keploy/keploy/tree/main/.github/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> **2** out of **3** committers have signed the CLA.✅ @Sonichigo✅ @Sarthak160❌ @DeepakchowdavarapuYou can retrigger this bot by commenting **recheck** in this Pull Request

recheck

File: .github/workflows/sample-run.yml [hunk 1]
```diff
@@ -22,4 +22,23 @@ jobs:
...
+
+      - name: Run Tests
+        run: |
+          go test -v ./...   # This runs all the tests in the project and provides detailed output.
+
+      - name: Display Test Logs
+        if: always()  #  runs regardless whether the tests pass or fail.
+        run: |
+          echo "Displaying logs from tests..."
+          go test -v ./... | tee test-output.log
```
Comment (PranshuSrivastava) [inline .github/workflows/sample-run.yml:36]: You don't need to use go test, you need to show the logs from the keploy test command that is given in the shell scripts.

**Review by PranshuSrivastava — CHANGES_REQUESTED** [review]

**Comment by Sarthak160** [comment]
@Deepakchowdavarapu Are you still working on this ?

**Comment by Hermione2408** [comment]
Closing this PR as it has been inactive for a long time. Feel free to reopen if needed.

**Pull request closed without merging**